docs/readme.md

```diff
@@ -1,9 +1,34 @@
 ---
 id: readme
-title: Welcome to FAST-DNA
+title: FAST-DNA
 sidebar_label: What is FAST-DNA
 ---
 
-FAST-DNA is a set of tooling, resources, and components used to accelerate building Web sites and applications.
+An unopinionated system of components, development tools, and utilities used à la carte or as a suite to build enterprise-grade websites and applications.
 
-Teams spend significant time building parts of the design system (colors, variable font, typography, grid, components, etc.) which can slow adoption of the latest design guidelines and related technologies. FAST-DNA solves this problem by providing tooling and resources to accelerate development with code that is performant and accessible. Many of the resources are written to be framework agnostic and will work with any component library and JavaScript framework (React, Angular, etc.). We also support a number of React-specific packages for UI generation.
\ No newline at end of file
+## Key features & benefits
+
+### unopinionated
```

Comment:
Should this be capitalized?

Reply:
It is now...

Reply:
Note, this page content is going to change to be the readme for the directory.